backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -12,7 +12,10 @@
         </strong></td>
           <td class="align-center">
             <span class="<%= @order.payments.failed.count > 0 ? 'state void' : 'state complete' %>">
-              <%= link_to "#{Spree.t 'payments_count', count: @order.payments.failed.count, default: pluralize(@order.payments.failed.count, Spree.t(:payment))}", spree.admin_order_payments_path(@order) %>
+              <%= link_to Spree.t(:payments_failed_count,
+                                  count: @order.payments.failed.count,
+                                  payments: Spree::Payment.model_name.human(count: @order.payments.failed.count)),
```

Comment:
`payments_failed_count` already makes it clear we are using payments, and we don't need to provide the model name to that translation.

The `:count` key is magic and will allow adding the `:one` and `:other` keys under `payments_failed_count`